includes/class-taxjar-tax-calculation.php

```diff
@@ -38,6 +38,18 @@ public function init_hooks() {
 		add_filter( 'wcs_new_order_created', array( $this, 'calculate_renewal_order_totals' ), 10, 3 );
 
 		add_action( 'woocommerce_order_after_calculate_totals', array( $this, 'maybe_calculate_order_taxes' ), 10, 2 );
+		add_action( 'woocommerce_checkout_create_order', array( $this, 'persist_cart_calculation_results_to_order'), 10, 1 );
+		add_action( 'woocommerce_checkout_create_subscription', array( $this, 'persist_recurring_cart_calculation_results_to_subscription'), 10, 4 );
+	}
+
+	public function persist_cart_calculation_results_to_order( $order ) {
+		$calculation_results = WC()->cart->tax_calculation_results;
+		$order->update_meta_data( '_taxjar_tax_result', $calculation_results );
+	}
+
+	public function persist_recurring_cart_calculation_results_to_subscription(  $subscription, $posted_data, $order, $recurring_cart ) {
```

Comment:
At first it looked like unused inputs here, but I'm guessing these "extra" params are necessary inputs when responding to the `woocommerce_checkout_create_subscription` hook?

Reply:
Yep, in order to access the recurring cart parameter we have to include all the ones that come before it. These are declared when the hook is fired so not anything we can change.